dev-python/gmpy/files/gmpy-2.1.0_beta5-test-input.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/test/runtests.py b/test/runtests.py
+index 5e5842d..7d64e52 100644
+--- a/test/runtests.py
++++ b/test/runtests.py
+@@ -81,7 +81,7 @@ if sys.version.startswith('3.1.'):
+     print("with Python 3.1. The doctest module in Python 3.2 and later does not")
+     print("have this issue.")
+     print()
+-    input("Press ENTER to continue.. ")
++    # input("Press ENTER to continue.. ")
```

Comment:
Don't add comments in patch, that's just inflating it unnecessarily.

Reply:
Removed the comment